.github/workflows/moodle-release.yml

```diff
@@ -1,64 +1,20 @@
 #
-# Whenever a new tag starting with "v" is pushed, add the tagged version
+# Whenever version.php is changed, add the latest version
 # to the Moodle Plugins directory at https://moodle.org/plugins
 #
 name: Releasing in the Plugins directory
 
 on:
   push:
-    tags:
-      - '*'
-
-  workflow_dispatch:
-    inputs:
-      tag:
-        description: 'Tag to be released'
-        required: true
-
-defaults:
-  run:
-    shell: bash
+    paths:
+      - 'version.php'
 
 jobs:
-  release-at-moodle-org:
-    runs-on: ubuntu-latest
-    env:
-      PLUGIN: auth_enrolkey
-      CURL: curl -s -u ${{ secrets.CURL_AUTH }}
-      ENDPOINT: https://next.moodle.org/webservice/rest/server.php
-      TOKEN: ${{ secrets.MOODLE_ORG_TOKEN }}
-      FUNCTION: local_plugins_add_version
-
-    steps:
-      - name: Call the service function
-        id: add-version
-        run: |
-          if [[ ! -z "${{ github.event.inputs.tag }}" ]]; then
-            TAGNAME="${{ github.event.inputs.tag }}"
-          elif [[ $GITHUB_REF = refs/tags/* ]]; then
-            TAGNAME="${GITHUB_REF##*/}"
-          fi
-          if [[ -z "${TAGNAME}" ]]; then
-            echo "No tag name has been provided!"
-            exit 1
-          fi
-          ZIPURL="https://api.github.com/repos/${{ github.repository }}/zipball/${TAGNAME}"
-          RESPONSE=$(${CURL} ${ENDPOINT} --data-urlencode "wstoken=${TOKEN}" \
-                                         --data-urlencode "wsfunction=${FUNCTION}" \
-                                         --data-urlencode "moodlewsrestformat=json" \
-                                         --data-urlencode "frankenstyle=${PLUGIN}" \
-                                         --data-urlencode "zipurl=${ZIPURL}" \
-                                         --data-urlencode "vcssystem=git" \
-                                         --data-urlencode "vcsrepositoryurl=${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}" \
-                                         --data-urlencode "vcstag=${TAGNAME}" \
-                                         --data-urlencode "changelogurl=${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}/commits/${TAGNAME}" \
-                                         --data-urlencode "altdownloadurl=${ZIPURL}")
-          echo "::set-output name=response::${RESPONSE}"
-
-      - name: Evaluate the response
-        id: evaluate-response
-        env:
-          RESPONSE: ${{ steps.add-version.outputs.response }}
-        run: |
-          jq <<< ${RESPONSE}
-          jq --exit-status ".id" <<< ${RESPONSE} > /dev/null
+  workflow_group_35_plus_release:
+    uses: catalyst/catalyst-moodle-workflows/.github/workflows/group-35-plus-release.yml@main
+    with:
+      plugin_name: auth_enrolkey
```

Comment:
can we infer the plugin name automatically?

ditto for the repository url. All aiming to make this file as minimal and zero config as possible.

Reply:
Hey @brendanheywood - I tried and I couldn't get the frankenstyle naming convention of the plugin name in reusable workflow file.  So I keep passing this from caller workflow. 

Removed other inputs "plugin_branch" and "plugin_repository_url".

Thanks,
Anu